kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -44,6 +44,9 @@
  * Document class override to ensure that the bank code is synchronized with the
  * payment method code.
  */
+// org.kuali.rice.kns.document.authorization.TransactionalDocumentPresentationController is deprecated
+// getReportable1099TransactionsFlag() and processAfterRetrieve() use unchecked Lists
+@SuppressWarnings({ "deprecation", "unchecked" })

```

Comment:
I too can't find an @Deprecated annotation anywhere. In case there's some old code somewhere that it's picking up, I've tried cleaning my Eclipse workspace, clearing out my maven repository, and several other things (short of deleting my entire workspace and downloading a fresh copy of the repo), the deprecation message is still there, so I think it's an Eclipse issue finding *something, somewhere*.